src/File/SimpleRelativePathHelper.php

```diff
@@ -15,10 +15,10 @@ public function __construct(string $currentWorkingDirectory)
 	public function getRelativePath(string $filename): string
 	{
 		if ($this->currentWorkingDirectory !== '' && strpos($filename, $this->currentWorkingDirectory) === 0) {
-			return substr($filename, strlen($this->currentWorkingDirectory) + 1);
+			return str_replace('\\', '/', substr($filename, strlen($this->currentWorkingDirectory) + 1));
 		}
 
-		return $filename;
+		return str_replace('\\', '/', $filename);
```

Comment:
do we need the same fix in other implementations of `RelativePathHelper` ?

Reply:
Not that I'm aware of. The one used for regular analysis already performs dirsep normalization.